src/index.coffee

```diff
@@ -18,7 +18,7 @@ convert = (proc) ->
       {fd: logHandle, filename: logPath} = results
 
       spawnOpts =
-        stdio: [ 'ignore', logHandle, logHandle ]
+        stdio: [ 'ignore', logHandle, logHandle, 'ipc' ]
```

Comment:
Nice, TIL!  The official node.js docs aren't entirely clear on how to set `ipc`.